x-pack/plugins/lens/public/xy_visualization/xy_config_panel/reference_line_panel.tsx

```diff
@@ -183,47 +175,49 @@ function getIconPositionOptions({ isHorizontal, axisMode }: LabelConfigurationOp
     defaultMessage: 'Right',
   });
   if (axisMode === 'bottom') {
-    const bottomOptions = [
-      {
-        id: `${idPrefix}above`,
-        label: isHorizontal ? rightLabel : topLabel,
-        'data-test-subj': 'lnsXY_referenceLine_markerPosition_above',
-      },
+    const options = [
       {
         id: `${idPrefix}below`,
         label: isHorizontal ? leftLabel : bottomLabel,
         'data-test-subj': 'lnsXY_referenceLine_markerPosition_below',
       },
+      autoOption,
+      {
+        id: `${idPrefix}above`,
+        label: isHorizontal ? rightLabel : topLabel,
+        'data-test-subj': 'lnsXY_referenceLine_markerPosition_above',
+      },
     ];
-    if (isHorizontal) {
-      // above -> below
-      // left -> right
-      bottomOptions.reverse();
-    }
-    return [...options, ...bottomOptions];
+    // if (isHorizontal) {
```

Comment:
leaving it for now while waiting for the feedback if we should go `top, auto, bottom` or `bottom, auto, top`